flask_init.txt

```diff
@@ -0,0 +1,12 @@
+from flask import Flask, make_response
+# from flask import Flask, Response
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+	return make_response('Hello World!', 200)
```

Comment:
This should return an HTML page, not just text.

Reply:
Done. Please see latest commit.